deploy/cephfs/kubernetes/csi-cephfsplugin-provisioner.yaml

```diff
@@ -148,10 +148,13 @@ spec:
           image: quay.io/cephcsi/cephcsi:canary
           args:
             - "--type=liveness"
+            - "--v=5"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--metricsport=8681"
             - "--metricspath=/metrics"
-            - "--polltime=60s"
+            - "--healthzport=9681"
+            - "--healthzpath=/healthz"
+            - "--polltime=15s"
```

Comment:
you can always pool the CSIDriver when you get a request to get live status.

Reply:
Maybe you mean probe? yes, we can always probe check.

As pointed other referenced projects are using 2sec and I feel 15 sec is a good time, but I will leave it up to the maintainers.